src/dataServices/dynamoDbBundleService.ts

```diff
@@ -37,16 +42,24 @@ export class DynamoDbBundleService implements Bundle {
 
     private dynamoDb: DynamoDB;
 
-    private maxExecutionTimeMs: number;
+    private readonly maxExecutionTimeMs: number;
 
     private static readonly dynamoDbMaxBatchSize = 25;
 
+    private readonly versionedLinks: any;
+
     // Allow Mocking DDB
-    constructor(dynamoDb: DynamoDB, supportUpdateCreate: boolean = false, maxExecutionTimeMs?: number) {
+    constructor(
+        dynamoDb: DynamoDB,
+        supportUpdateCreate: boolean = false,
+        maxExecutionTimeMs?: number,
+        versionedLinks?: Record<string, string[]>,
```

Comment:
Can you add comments to this constructor now versionedLinks isn't clear what it is

Reply:
Add the new constructor param as:
```suggestion
        { versionedLinks }: { versionedLinks?: Record<string, string[]> } = {},
```
Reasons for this are explained on https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/pull/86#discussion_r656733648